mne/time_frequency/csd.py

```diff
@@ -255,3 +221,187 @@ def compute_epochs_csd(epochs, mode='multitaper', fmin=0, fmax=np.inf,
                                              frequencies=frequencies[i],
                                              n_fft=n_fft))
         return csds
+
+
+@verbose
+def csd_array(X, sfreq, mode='multitaper', fmin=0, fmax=np.inf,
+              fsum=True, n_fft=None, mt_bandwidth=None,
+              mt_adaptive=False, mt_low_bias=True, verbose=None):
+    """Estimate cross-spectral density from an array.
+
+    .. note:: Results are scaled by sampling frequency for compatibility with
+              Matlab.
+
+    Parameters
+    ----------
+    X : array-like, shape (n_replicates, n_series, n_times)
```

Comment:
@Eric89GXL @agramfort Since this is the function that is generally applicable shouldn't we avoid epoch-specific language? I agree with Eric that providing explicit examples in the docstring would be helpful, but I think it's more confusing to use `n_channels` when this function could be used for source-space analysis where you don't have channels. By using `n_channels` we're catering to users who are analyzing `Epoch`s, but there is an explicit function for that analysis that uses epoch-specific terminology. Does this sound reasonable to you? If you really feel strongly about using the epoch-specific terminology I'll use it, but it does make me uncomfortable having the generic version of the function use vocabulary from a specific use-case.